### PR TITLE
fix: kosmosctl add floater logic and remove linkctl

### PR DESCRIPTION
**Pull request by ONE7live**

### CODE REVIEW

File: pkg/kosmosctl/floater/doctor.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+					return fmt.Errorf("%q does not take any arguments, got %q", cmd.CommandPath(), args)
+				}
+			}
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&o.Namespace, "namespace", "n", util.DefaultNamespace, "Kubernetes namespace.")
+	flags.StringVarP(&o.ImageRepository, "image-repository", "r", "ghcr.io/kosmos-io/clusterlink", "Image repository.")
```
Comment (hanweisen) [inline pkg/kosmosctl/floater/doctor.go]: modify default repo to ghcr.io/kosmos-io

File: pkg/kosmosctl/floater/doctor.go [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+				if len(arg) > 0 {
+					return fmt.Errorf("%q does not take any arguments, got %q", cmd.CommandPath(), args)
+				}
+			}
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&o.Namespace, "namespace", "n", util.DefaultNamespace, "Kubernetes namespace.")
```
Comment (hanweisen) [inline pkg/kosmosctl/floater/doctor.go]: is kubernetes namespace right?

Comment (ONE7live) [reply pkg/kosmosctl/floater/doctor.go]: Kosmos namespace? what do you think?

File: pkg/kosmosctl/floater/doctor.go (hunk 1 above)
Comment (ONE7live) [reply pkg/kosmosctl/floater/doctor.go]: ok~I'm done

**Pull request merged**